topics/socket-io/index.md

```diff
@@ -0,0 +1,12 @@
+---
+topic: socket-io
+display_name: "Socket.IO"
+related: nodejs
+logo: socketio.png
+url: https://socket.io
+github_url: https://github.com/socketio/socket.io
+wikipedia_url: https://en.wikipedia.org/wiki/Socket.IO
+short_description: "Socket.IO is a JavaScript library for realtime web applications"
```

Comment:
I did a quick check and the other short descriptions use punctuation. Let's add some here to be consistent 😄 

```suggestion
short_description: "Socket.IO is a JavaScript library for realtime web applications."
```

Reply:
My bad 😛